hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/executorservice/ExecutorServiceCancelOnAddressMessageTask.java

```diff
@@ -35,14 +31,10 @@ public ExecutorServiceCancelOnAddressMessageTask(ClientMessage clientMessage, No
     }
 
     @Override
-    protected InvocationBuilder createInvocationBuilder() throws UnknownHostException {
-        final InternalOperationService operationService = nodeEngine.getOperationService();
-        final String serviceName = DistributedExecutorService.SERVICE_NAME;
-        CancellationOperation op = new CancellationOperation(parameters.uuid, parameters.interrupt);
-        return operationService.createInvocationBuilder(serviceName, op, parameters.address);
+    protected Operation prepareOperation() {
+        return new CancellationOperation(parameters.uuid, parameters.interrupt);
```

Comment:
What about `parameters.address`? I don't see it's used anywhere.

Reply:
Yes, you are right. I changed `ExecutorServiceCancelOnAddressMessageTask` to extend the `AbstractAddressMessageTask` and overwrote `getAddress()` method. Removed the `AbstractExecutorServiceCancelMessageTask` as well and carried the methods to `ExecutorServiceCancelOnPartitionMessageTask`.